src/simulator_core/entities/network_controller.py

```diff
@@ -13,9 +13,21 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-""" NetworkController entity."""
+"""NetworkController entity."""
 
 
 class NetworkController:
+    """Class to store the network controller."""
+
     def __init__(self):
```

Comment:
https://mypy.readthedocs.io/en/stable/error_code_list2.html#check-that-every-function-has-an-annotation-no-untyped-def

Even an init needs to have a return if there are no arguments supplied!

Reply:
Code stills needs to be developed then it can be updated

Reply:
#22  is already for this question